hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -342,12 +343,10 @@ void testListStatusWithIntermediateDirWithECEnabled()
     String key = "object-dir/object-name1";
 
     // write some test data into bucket
-    try (OzoneOutputStream outputStream = objectStore.getVolume(volumeName).
-            getBucket(bucketName).createKey(key, 1,
-                    new ECReplicationConfig("RS-3-2-1024k"),
-                    new HashMap<>())) {
-      outputStream.write(RandomUtils.nextBytes(1));
-    }
+    byte[] bytes = new byte[1];
+    random.nextBytes(bytes);
+    TestDataUtil.createKey(objectStore.getVolume(volumeName).getBucket(bucketName), key,
+        new ECReplicationConfig("RS-3-2-1024k"), bytes);
```

Comment:
Why don't you keep `RandomUtils.nextBytes(1)`?
```suggestion
    TestDataUtil.createKey(objectStore.getVolume(volumeName).getBucket(bucketName), key,
        new ECReplicationConfig("RS-3-2-1024k"), RandomUtils.nextBytes(1));
```

Reply:
`RandomUtils.secure().randomBytes(1)`

Reply:
I think we don't need to use secure here, or you just want to avoid the `deprecated` warning?